.travis.yml

```diff
@@ -1,11 +1,17 @@
 language: elixir
 elixir:
-  - 1.2.3
+  - 1.1.1
+  - 1.2.6
+  - 1.3.2
 otp_release:
-  - 18.2
+  - 18.3
+  - 19.0
 sudo: false
 env:
   - MIX_ENV=test STRIPE_SECRET_KEY=non_empty_secret_key_string
 script: mix coveralls.travis
 after_script:
   - mix inch.report
+exclude:
+  - otp_release: 19.0
+    elixir: 1.1.1
```

Comment:
@gmile I guess my question though is whether we want `exclude` when we're listing it above? Wouldn't we just not list it above and then `exclude` isn't necessary?